changes/626.feature.md

```diff
@@ -0,0 +1 @@
+Refactoring `decrypt_payload()` as a middleware so that applied to other request handlers
```

Comment:
I suggest to write in an imperative form. Maybe we can say a verb "Refactor" than "Refactoring".
Also, it would be great if it is clearly specified which is applied to other request handlers.